mergify_engine/actions/review.py

```diff
@@ -102,25 +102,15 @@ async def run(
                 "body"
             ] = f"Pull request automatically reviewed by Mergify: {self.config['type']}"
 
-        reviews = list(
-            reversed(
-                list(
-                    filter(
-                        lambda r: r["user"]["login"] is not review_user,
-                        await ctxt.reviews,
-                    )
+        reviews = reversed(
+            list(
+                filter(
+                    lambda r: r["user"]["login"] == review_user,
```

Comment:
did you mean `==` or `!=`?

Reply:
`==` I have rewrite the test to prove it ;) and even check the new test fail with the old code.